test/verify/machineslib.py

```diff
@@ -1040,6 +1046,10 @@ def release_target(target):
         # avoid error noise about resources getting cleaned up
         b.logout()
 
+        # AppArmor doesn't like the non-standard path for our storage pools
+        if m.image in ["debian-testing"]:
+            self.allow_journal_messages('.* type=1400 .* apparmor="DENIED" operation="open" profile="libvirtd.* name="%s .*denied_mask="r"' % self.tmp_storage)
```

Comment:
Not sure if there are more similar problems, or this needs to be more robust, but it does not cover:
```
audit: type=1400 audit(1584487308.478:83): apparmor="DENIED" operation="open" profile="libvirt-364cd049-96ba-47a2-8c43-94cbfd361f00" name="/mnt/libvirt/vm_one/mydiskofpoolone_temporary" pid=49479 comm="qemu-system-x86" requested_mask="r" denied_mask="r" fsuid=64055 ouid=64055
```